contrib/seeds/generate-seeds.py

```diff
@@ -46,11 +46,11 @@ def name_to_bip155(addr):
     '''Convert address string to BIP155 (networkID, addr) tuple.'''
     if addr.endswith('.onion'):
         vchAddr = b32decode(addr[0:-6], True)
-        if len(vchAddr) == 10:
-            return (BIP155Network.TORV2, vchAddr)
-        elif len(vchAddr) == 35:
-            assert(vchAddr[34] == 3)
+        if len(vchAddr) == 35:
+            assert vchAddr[34] == 3
             return (BIP155Network.TORV3, vchAddr[:32])
+        elif len(vchAddr) == 10:
+            return (BIP155Network.TORV2, vchAddr)
```

Comment:
This re-ordering change looks unnecessary now. The diff would be smaller when you revert this 4 line change. :-)

Reply:
Most frequent path first.